content/blog/2019/10/2019-10-21-thinking-about-jenkins-security.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Thinking About Jenkins Security - DevOps World 2019"
```

Comment:
```suggestion
title: "Thinking About Jenkins Security - DevOps World 2019"
```

Are you sure you want to forget about "Jenkins World" in the jenkins.io blog? :P

Reply:
Resolved in e02460f75da1eded63e0fc1299ecfd53f93f6a8d